hail/src/test/scala/is/hail/asm4s/ASM4SSuite.scala

```diff
@@ -13,37 +14,37 @@ import scala.language.postfixOps
 
 trait Z2Z { def apply(z:Boolean): Boolean }
 
-class ASM4SSuite extends TestNGSuite {
+class ASM4SSuite extends HailSuite {
```

Comment:
This feels not great, but I wanted a single source of truth for `shouldWriteIRFiles`